sdk/storage/storage-blob/test/node/blobclient.spec.ts

```diff
@@ -38,6 +38,7 @@ describe("BlobClient Node.js only", () => {
   let blockBlobClient: BlockBlobClient;
   const content = "Hello World";
   const tempFolderPath = "temp";
+  const timeoutForLargeFile = 20 * 60 * 1000;
```

Comment:
I believe a similar(or same) timeout is used under `node/highlevel.spec.ts`

Reply:
Let's use a large timeout per suite as we may want to use different timeout value. So won't refactor it as a common constants for now.